src/metabase/models/setting.clj

```diff
@@ -406,6 +411,13 @@
   set to true when settings are being written directly via /api/setting endpoints."
   false)
 
+(defn- has-feature?
+  [feature]
+  (u/ignore-exceptions
+   (classloader/require 'metabase.public-settings.premium-features))
+  (let [has-feature?' (resolve 'metabase.public-settings.premium-features/has-feature?)]
```

Comment:
maybe we can just `def` this from top-level? no need to make it a wrapper I think